go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/metal-toolbox/bioscfg
 go 1.23.1
 
 require (
-	github.com/bmc-toolbox/bmclib/v2 v2.3.4
+	github.com/bmc-toolbox/bmclib/v2 v2.3.5-0.20241111155613-31918a066b74
```

Comment:
why not cut a release of bmclib?

Reply:
I just dont want to deal with the same stuff as last time. I made sure to pull this commit from main, instead of the branch, so this wont result in issues with the commit no longer existing.